src/UI/Actions.hs

```diff
@@ -1321,6 +1322,19 @@ fileBrowserToggleFile =
     fb' <- lift $ FB.maybeSelectCurrentEntry fb
     assign (asFileBrowser . fbEntries) fb'
 
+-- | Search related mails sent from the same authors.
+--
+searchRelated :: Action 'Threads 'ListOfThreads ()
+searchRelated = Action ["search related mail"] $ do
+  authors <- preuse (asThreadsView . miThreads . listList . to L.listSelectedElement . _Just . _2 . _2 . thAuthors . traverse)
+  case firstOf traverse authors of
+    Nothing -> runExceptT (throwError (ProcessError "No authors availabe to perform search"))
```

Comment:
Not sure if `ProcessError` is suitable.  That constructor is used for subprocess errors.

Reply:
You're right.. I'm getting sloppy. Will change.

Reply:
I named it `InvalidQueryError`.. I'm not sure whether that's a bit too specific to the query. The error is perhaps more UI, than query related. If you have a better word in your mind - all ears :)

Reply:
`InvalidQueryError` is fine with me.  Merging.